bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/CheckBoxImpl.java

```diff
@@ -56,13 +56,13 @@ public class CheckBoxImpl extends AbstractOptionsFieldImpl implements CheckBox {
     private String uncheckedValue;
 
     @ValueMapValue(injectionStrategy = InjectionStrategy.OPTIONAL)
-    private Boolean enableUncheckedValue;
+    private Boolean preserveUncheckedStateValue;
```

Comment:
This might break existing forms, you can keep the property name as it was earlier.